CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/text/TextPluginInteraction.java

```diff
@@ -60,6 +60,11 @@ public void setProgress(int currentStep, int totalSteps, String message, boolean
         currentMessage = message;
         LOGGER.log(Level.INFO, "currentStep={0} totalSteps={1} message={2}", new Object[]{currentStep, totalSteps, message});
     }
+    
+    @Override
+    public void setProgress(int currentStep, int totalSteps, boolean cancellable) throws InterruptedException {
+        //LOGGER.log(Level.INFO, "currentStep={0} totalSteps={1} message={2}", new Object[]{currentStep, totalSteps, message});
```

Comment:
are you able to remove this commented code, or alternatively uncomment and use (will need to remove message part first though)